README.md

```diff
@@ -6,52 +10,71 @@ GreynirSeq is a natural language parsing toolkit for Icelandic focused on sequen
 
 The modeling part (nicenlp) of GreynirSeq is built on top of the excellent [fairseq](https://github.com/pytorch/fairseq) from Facebook (which is built on top of pytorch).
 
-GreynirSeq is licensed under the GNU GPLv3 license unless otherwise stated at the top of a file.
+GreynirSeq is licensed under the GNU AFFERO GPLv3 license unless otherwise stated at the top of a file.
 
-## What's on the horizon?
-* Cleanup and configuration of data / model loading -- currently unavailable for download
-* More fine tuning tasks for Icelandic
-* General cleanup and CI config
+**What's new?**
+* This repository!
+* An Icelandic RoBERTa model, **IceBERT** finetuned for NER and POS tagging.
+
+**What's on the horizon?**
+* More fine tuning tasks for Icelandic, constituency parsing and grammatical error detection
 * Icelandic - English translation example
-* Improved documentation and examples for training and preprocessing pipelines
 
-## What's new?
-* This repository!
-* An Icelandic RoBERTa model, **IceBERT** with NER, POS tagging and constituency parsing fine tuning options.
-* Simple Docker setup to serve models
-* NER pre and post processing for NMT corpora
+---
 
-## Neural Icelandic Language Processing - NIceNLP
+Be aware that usage of the CLI or otherwise downloading model files will result in downloading of **gigabytes** of data.
 
-### IceBERT
+## Features
 
-IceBERT is an Icelandic language model.
+### TL;DR give me the CLI
+
+The `greynirseq` CLI interface can be used to run state-of-the-art POS and NER tagging for Icelandic. Run `pip install greynirseq && greynirseq -h` to see what options are available.
+
+#### POS
+
+``` bash
+❯ pip install greynirseq
+❯ echo "Systurnar Guðrún og Monique átu einar um jólin á McDonalds ." | greynirseq pos --input -
```

Comment:
Kannski líka bæta við dæmi sem sýnir að cli styður líka skrár.

Reply:
This is the TL;DR, `greynirseq -h` is mentioned above